### DISCUSSION TITLE
Specifying language to be detected

### DISCUSSION THREAD

**Discussion by yumikim381** [mindee/doctr#1516] [General]
Thank you for your amazing work!
is there a way to specify language that should be OCRed ?

**Comment by DanielVegaVega** [comment]
Also interested on this, I am pretty new to OCR models.

**Comment by felixdittrich92** [comment]
Hi @yumikim381 @DanielVegaVega :wave:,

All models are pretrained on french vocab.

Option 1: 
I have fine tuned a multilingual model a while ago you could try it:
https://huggingface.co/Felix92/doctr-torch-parseq-multilingual-v1
model vocab includes: `english & french & german & italian & spanish & portuguese & czech & polish & dutch & norwegian & danish & finnish & swedish & §`

Option 2:
or you fine tune your own model:
https://mindee.github.io/doctr/using_doctr/custom_models_training.html

Best regards,
Felix
